v2/emailpassword/sidebars.js

```diff
@@ -220,7 +220,15 @@ module.exports = {
             "advanced-customizations/frontend-hooks/handle-event",
             "advanced-customizations/frontend-hooks/redirection-callback"
           ],
-        }
+        },
+        {
+          type: 'category',
```

Comment:
I would say that put this example inside advanced customisation > "Examples" > "Using localstorage instead of cookies"